raiden/routing.py

```diff
@@ -122,6 +92,28 @@ def get_best_routes(
             )
             continue
 
+        try:
+            length = networkx.shortest_path_length(
+                token_network.network_graph.network,
+                partner_address,
+                to_address,
+            )
+            heappush(neighbors_heap, (partner_address, length, nonrefundable))
```

Comment:
I guess `partner_address` should be the last item in the tuple.

Reply:
right, will do!